### PR TITLE
First pass at scraping Blockly Games

### PR DESCRIPTION
**Pull request by divad12**
Source: https://blockly-games.appspot.com

7 puzzle games to teach programming using drag-and-drop blocks. Open source project by Google translated into 49 languages. 

We make HTML5 apps out of each game.

Scrapes most of the languages except four because we currently don't have them in `le-utils` (it's a TODO to fix).

### CODE REVIEW

File: chef.py [hunk 1]
```diff
@@ -0,0 +1,459 @@
...
+    with WebDriver("https://blockly-games.appspot.com", delay=1000) as driver:
+        for option in driver.find_elements_by_css_selector('#languageMenu option'):
+            blockly_language_code = option.get_attribute('value')
+            language_title = option.text
+            le_language_code = blockly_language_code
+
+            # There are some obscure languages that Blockly supports that we
+            # don't yet know about in le-utils. Skip those for now.
+            # TODO(davidhu): Add these language codes to le-utils
+            if le_language_code in ['hrx', 'pms', 'sco', 'be-tarask']:
```
Comment (ivanistheone) [inline chef.py:116]: I'm working on a PR to add `sco` to `le-utils` soon, and I can add the others as well.

File: chef.py [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+        try:
+            response = sess.get(url, headers=headers, timeout=timeout, *args, **kwargs)
+            break
+        except (requests.exceptions.ConnectionError, requests.exceptions.ReadTimeout) as e:
+            retry_count += 1
+            print("Error with connection ('{msg}'); about to perform retry {count} of {trymax}."
+                  .format(msg=str(e), count=retry_count, trymax=max_retries))
+            time.sleep(retry_count * 1)
+            if retry_count >= max_retries:
+                return Dummy404ResponseObject(url=url)
```
Comment (ivanistheone) [inline chef.py:439]: If you want this not to be dangling, here it is copy-pasted from https://github.com/learningequality/sushi-chef-ck12/blob/master/chef.py#L77

```
class Dummy404ResponseObject(requests.Response):
    def __init__(self, url):
        super(Dummy404ResponseObject, self).__init__()
        self._content = b""
        self.status_code = 404
        self.url = url
```

**Review by ivanistheone — APPROVED** [review]

File: chef.py (hunk 1 above)
Comment (divad12) [reply chef.py:116]: Awesome!

**Comment by divad12** [comment]
Thank you for the review Ivan! :)

**Pull request merged**